src/components/Common/Tabs/Tabs.js

```diff
@@ -71,6 +71,7 @@ export const TabsItem = ({
   const { switchTab, selectedTab, lastTab } = useContext(TabsContext);
   const [currentTitle, setCurrentTitle] = useState(title);
   const [renameMode, setRenameMode] = useState(false);
+  const [hover, setHover] = useState(false);
```

Comment:
do we need hover as a state? as I see it is used only for styling

Reply:
the problem here is that a dropdown is not inside the button, so when you hover over a tab – three dots are shown, you can click on it to open a dropdown, but as soon as you move your cursor to the dropdown – menu button disappears